test/send_recv.c

```diff
@@ -214,18 +214,20 @@ static int do_send(void)
 	ret = io_uring_wait_cqe(&ring, &cqe);
 	if (cqe->res == -EINVAL) {
 		fprintf(stdout, "send not supported, skipping\n");
-		close(sockfd);
-		return 0;
-	}
+	    goto err;
+    }
```

Comment:
Please use tabs as the indentation. Don't use spaces.

Reply:
Done